g11n-ws/tools/tool-cldr-extractor/build.gradle

```diff
@@ -31,14 +31,16 @@ repositories {
 }
 
 dependencies {
-    compile group: 'commons-collections', name: 'commons-collections', version: '3.2.2'
-    testCompile group: 'junit', name: 'junit', version: '4.+'
-    compile group: 'com.googlecode.json-simple', name: 'json-simple', version: '1.1.1'
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.11.2'
-    compile group: 'commons-io', name: 'commons-io', version: '2.4'
-    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.26'
-    testCompile group: 'org.slf4j', name: 'slf4j-log4j12', version: '1.7.26'
-    testCompile group: 'log4j', name: 'log4j', version: '1.2.17'
+    implementation group: 'commons-collections', name: 'commons-collections', version: '3.2.2'
+    implementation('com.googlecode.json-simple:json-simple:1.1.1'){
+          exclude group: 'junit'
```

Comment:
why need to exclude junit here?

Reply:
because json-simple 1.1.1 depended junit jar version conflict with single depended junit jar version